client/layout/masterbar/logged-in.jsx

```diff
@@ -149,9 +149,9 @@ class MasterbarLoggedIn extends React.Component {
 					tooltip={ translate( 'Update your profile, personal settings, and more' ) }
 					preloadSection={ this.preloadMe }
 				>
-					<Gravatar user={ this.props.user } alt="Me" size={ 18 } />
+					<Gravatar user={ this.props.user } alt={ translate( 'My Profile' ) } size={ 18 } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **73** times:
`translate( 'My Profile' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?